plugins/ssr/ssrvernac.mlg

```diff
@@ -300,6 +300,35 @@ VERNAC COMMAND EXTEND HintView CLASSIFIED AS SIDEFF
           Ssrview.AdaptorDb.declare k hints }
 END
 
+(** Search compatibility *)
+
+{
+
+  let warn_search_moved_enabled = ref true
+  let warn_search_moved = CWarnings.create ~name:"ssr-search-moved"
+      ~category:"deprecation" ~default:CWarnings.Enabled
```

Comment:
Sorry for noticing that late, but this category does not exist. The following one does:
```suggestion
      ~category:"deprecated" ~default:CWarnings.Enabled
```

Reply:
This is an area where we would IMHO gain from having a datatype + constructors.

Reply:
Fixed.

Reply:
>This is an area where we would IMHO gain from having a datatype + constructors.

Yes, I noticed this and I'm currently preparing a PR doing exactly this.